workspace-grid@daemonl/files/metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-    "cinnamon-version": ["2.0", "2.1", "2.2"],
+  "cinnamon-version": ["2.0", "2.1", "2.2", "3.2"],
```

Comment:
Since its working on 3.2, can you add 2.4, 2.6, 2.8, and 3.0 as well? It probably doesn't have compatibility issues if its still working on 3.2.